test/api/test_gpu.cpp

```diff
@@ -1,14 +1,26 @@
 #include <numeric>
 #include <migraphx/migraphx.h>
 #include <migraphx/migraphx.hpp>
+#include <migraphx/compile_options.hpp>
 #include "test.hpp"
 
+TEST_CASE(compile_options_api_test)
```

Comment:
Can you move this test to a different C++ file? This way we dont need to change everything to use `migraphx::api::compile_options` in this test file.

Reply:
Done.